packages/gasket-plugin-nextjs/test/config.test.js

```diff
@@ -239,31 +226,31 @@ describe('createConfig', () => {
         }
       }, { isServer: false, defaultLoaders: {}, dev: true, config: nextConfig });
 
-      assume(resultConfig.resolve.alias['@gasket/example']).equals(__filename);
-      assume(resultConfig.resolve.alias['@gasket/what']).equals('@gasket/example');
+      expect(resultConfig.resolve.alias['@gasket/example']).toEqual(__filename);
+      expect(resultConfig.resolve.alias['@gasket/what']).toEqual('@gasket/example');
     });
 
     describe('built-ins', () => {
 
       it('configures SASS loader', () => {
         result = config.webpack(webpackConfig, data);
 
-        assume(result.module.rules.some(rule => rule.test.test('bogus.scss')));
-        assume(result.module.rules.some(rule => rule.test.test('bogus.sass')));
+        expect(result.module.rules.some(rule => rule.test.test('bogus.scss'))).toBeFalsy();
+        // expect(result.module.rules.some(rule => rule.test.test('bogus.sass')));
```

Comment:
Still working on this expect?

Reply:
No forgot to delete it. This was doing nothing previously IMO.

Reply:
It had no assertions